src/app/dialogs/rna-dialog/rna-dialog.component.html

```diff
@@ -54,17 +54,15 @@
                 <div class="row last-content-row">
                     <p>
                         RNA-seq data from these studies was processed through a
-                        <span class="link auto-width no-bold" (click)="openWindow('https://www.synapse.org/#!Synapse:syn9702085')">harmonized computational pipeline</span>&nbsp;
+                        <span class="link auto-width no-bold" (click)="openWindow('https://adknowledgeportal.synapse.org/Explore/Studies?Study=syn9702085')">harmonized computational pipeline</span>&nbsp;
                         to reveal overall expression levels, measure differential expression between AD cases and controls, and describe the gene coexpression network.
                     </p>
                 </div>
                 <div class="row last-content-row">
                     <p>
-                        The data is publicly available in
-                        <span class="link auto-width no-bold" (click)="openWindow('https://www.synapse.org/')">Synapse</span>
-                        through these links:
-                        <span class="link auto-width no-bold" (click)="openWindow('https://www.synapse.org/#!Synapse:syn14237651')">RNA-seq differential expression summary statistics;</span>&nbsp;
-                        <span class="link auto-width no-bold" (click)="openWindow('https://doi.org/10.1101/510420')">RNA-seq network coexpression analysis</span>.
+                        The data is publicly available through these links:
+                        <span class="link auto-width no-bold" (click)="openWindow('https://adknowledgeportal.synapse.org/Explore/Studies?Study=syn9702085')">RNA-seq differential expression summary statistics;</span>&nbsp;
```

Comment:
is ';' after statistics intentional? If so - looks good!

Reply:
I think it's a question for Anna. I can check with her but the ; is currently on prod also.